core/src/main/scala/kafka/log/Log.scala

```diff
@@ -2419,7 +2419,7 @@ class Log(@volatile private var _dir: File,
 
       // replace old segment with new ones
       info(s"Replacing overflowed segment $segment with split segments $newSegments")
-      replaceSegments(newSegments.toList, List(segment), isRecoveredSwapFile = false)
+      replaceSegments(newSegments.toList, List(segment))
```

Comment:
@guozhangwang Is this ok? I am not familiar with this part of the code. If `false` the default anyway?

Reply:
LGTM, since the method definition also has `isRecoveredSwapFile: Boolean = false` by default.

Reply:
Thanks @guozhangwang for verifying.